src/components/TimeSeriesCharts/TimeSeriesChartHeader.tsx

```diff
@@ -144,6 +186,16 @@ export default function TimeSeriesChartHeader({
             <CommentIcon fontSize="inherit" />
           </ToggleButton>
         </Tooltip>
+        <Tooltip arrow title={i18n.get("Toggle using a bar chart")}>
```

Comment:
I think  "Toggle showing a bar chart" can be more meaningful.

Reply:
Agreed. Will even be more clear: "Toggle between showing a bar or line chart"

Reply:
But for line chart we already have button on left so I think its fine to write toggle showing a bar chart